MicrosoftActiveDirectory/CHANGELOG.md

```diff
@@ -11,4 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Add for the first time microsotf AD actions
\ No newline at end of file
+- Add for the first time microsotf AD actions
+
+## 2023-11-29 - 1.1.0
+
+### Update
+
+- Update LDAP query
+- Add some tests
```

Comment:
The order of entry is from the most recent change at the top, and the older one at the bottom.
Can you reverse the entries?

Reply:
Yes thank you for this remark